.npmignore

```diff
@@ -2,5 +2,3 @@ coverage/
 .nyc_output/
 .vscode/
 test/
-
-yarn.lock
```

Comment:
should we replace this with `package-lock.json` since, as you note, it won't be respected by downstream consumers?

Reply:
For whatever reason, I don't typically see lock files getting ignored, so I figured we could just leave them alone.

Reply:
I suspect that's more due to not thinking about it than a cogent argument ;)

but agreed it's not a big deal